plugin/pkg/shoot/validator/admission.go

```diff
@@ -327,6 +327,7 @@ func (v *ValidateShoot) Admit(ctx context.Context, a admission.Attributes, _ adm
 	allErrs = append(allErrs, validationContext.validateRegion()...)
 	allErrs = append(allErrs, validationContext.validateProvider(a)...)
 	allErrs = append(allErrs, validationContext.validateAdmissionPlugins(a, v.secretLister)...)
+	allErrs = append(allErrs, validationContext.validateOIDCConfig(a)...)
```

Comment:
Nit: I would rather move the `validateOIDCConfig` invocation in the `validateKubernetes` func. For example in the static validation `ValidateShootSpec` -> `validateKubernetes` -> `ValidateKubeAPIServer` (the final funcs that validates the oidc fields).